tests/test_synth.py

```diff
@@ -0,0 +1,101 @@
+"""
+Tests for torch synths
+"""
+
+
+import pytest
+import torch.nn
+import torch.tensor as T
+
+import torchsynth.globals
+import torchsynth.module as synthmodule
+import torchsynth.parameter
+import torchsynth.synth
+from torchsynth.parameter import ModuleParameter
+
+
+class TestTorchSynth:
```

Comment:
this should be renamed to TestAbstractSynth

Reply:
done